lbnl-nhc.spec.in

```diff
@@ -43,7 +43,7 @@
 Summary: LBNL Node Health Check
 Name: %{name}
 Version: %{version}
-Release: %{release}%{?dist}.ug
+Release: %{release}.ug.1
```

Comment:
dist is already included in release. we now have a double el8 in `lbnl-nhc-1.4.3-0.155.g4343.el8.el8.ug.noarch`

Reply:
and yes, that is a UGent change :smile:

Reply:
not sure what yum will think of it though. if it will be regarder as higher.

Reply:
@itkovian opinions?

Reply:
I would ditch the dist then.